### DISCUSSION TITLE
Unable to Pick master branch's success npm build in order to implement this in React Native?

### DISCUSSION THREAD

**Discussion by MuhammadRafeh** [software-mansion/react-native-reanimated#2376] [Q&A] [answered]
According to docs! In order to install reanimated 2, we have to pick master branch build. But what I found all the failed builds on master branch. So how can I pick it??? Or their is another way to setup this in react native???

```
Using master branch builds#
To use Reanimated 2 built from the master branch:

go to the "Build npm package" workflow in Reanimated repository
select latest build and download react-native-reanimated-2.0.0-alpha.tgz artifact
run tar zxvf react-native-reanimated-2.0.0-alpha.tgz.zip to unpack zip (or unpack it manually)
run yarn add file:react-native-reanimated-2.0.0-*.tgz to install the package
run cd android && ./gradlew clean
```

**Comment by MuhammadRafeh** [comment, marked as answer]
Yes, we should skip this step (Using master branch builds) while going through installation process because this instruction is subheader of "Installing the package" and has "Reanimated 2 in Expo" as a sibling so it's one way of installing the package.

**Reply by terrysahaidak** [reply]
It's not always possible to build it from master.

But! There is a very recent release of Reanimated available. Use official Alpha release if you need latest features.

**Reply by MuhammadRafeh** [reply]
Ok. It's mean we can also download this with releases! Thanks for help.

**Reply by jgonet** [reply]
Builds from master are meant for reproducing bugs mostly. You probably want to install reanimated from the npm registry.

**Reply by MuhammadRafeh** [reply]
@jakub-gonet So, it mean I should skip this step while going through installation process (Using master branch builds).

**Reply by jgonet** [reply]
Yes, this instruction is subheader of "Installing the package" and has "Reanimated 2 in Expo" as a sibling so it's one way of installing the package.

**Comment by MuhammadRafeh** [comment]
@jakub-gonet Thank u so much brother for helping me out.
